bin/git_rails

```diff
@@ -153,7 +153,7 @@ if [ ! -z "$migrations" ]; then
 
   # RUN THE MIGRATIONS (AND TEST PREPARE)
   {
-    echo "$migrate_commands" | bundle exec rails c
+    echo "$migrate_commands" | bundle exec rails runner /dev/stdin
```

Comment:
Is this necessary for the subdirectory feature?

Reply:
No. I changed this because rails console prints some tty-related errors when not connected to a terminal (such as when running in a git hook). Should I make a separate PR for this?

Reply:
A separate pr  would be awesome.

Reply:
I have merged it in #12 
This pr should be rebased.

Reply:
Thanks, rebased.